app/policies/organizations/adopter_foster_adopted_pet_policy.rb

```diff
@@ -0,0 +1,15 @@
+class Organizations::AdopterFosterAdoptedPetPolicy < ApplicationPolicy
```

Comment:
Ah yes this is close to what I had in mind with this comment https://github.com/rubyforgood/pet-rescue/pull/910/files#r1720837255...I should have looked further down :) 

Still, we should be querying Match instead of AdopterApplication. 
Also, this name `Organizations::AdopterFosterAdoptedPetPolicy` is duplicating a common module name in this app `AdopterFosterer` so let's add that directory so we have `Organizations::AdopterFosterer::MatchPolicy` (I think the policy names ideally mirror the model names, right?)